interview/password.py

```diff
@@ -15,5 +15,38 @@ def generate_password() -> str:
     # https://docs.python.org/3/library/random.html#random.randint
 
     # TODO implement generate_password function
+    import random 
+
+    password = ''
+
+    # pull 5 random letters
+    alphabet = 'abcdefghijklmnopqrstuvwxyz'
+    r = alphabet[random.randint(0, len(alphabet) - 1)]
```

Comment:
this is an acceptable implementation -- just be aware that when using random this way all passwords contain duplicates (ex: 'aaaaa1111!')